File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int s , mx = INT_MIN;
        for(int i = 0 ; i < accounts.size() ; i++){
            s = 0;
            for(int j = 0 ; j < accounts[i].size() ; j++){
                s += accounts[i][j];
                mx = max(s,mx);
            }
        }
        return mx;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        mx = -1
        for i in range(len(accounts)):
            s = 0
            for j in range (len(accounts[i])):
                s += accounts[i][j]
                mx = max(s,mx)
        return mx
        